React360/js/Compositor/Video/BrowserVideoPlayer.js

```diff
@@ -131,13 +131,13 @@ export default class BrowserVideoPlayer implements VideoPlayer {
   }
 
   play() {
-    this._element.play();
     this._playing = true;
+    this._element.play();
```

Comment:
shouldn't this line be `return this._element.play();` like the other lines that start with `return`?